internal/api/ui/login/router.go

```diff
@@ -14,6 +14,7 @@ const (
 	EndpointExternalLogin                 = "/login/externalidp"
 	EndpointExternalLoginCallback         = "/login/externalidp/callback"
 	EndpointExternalLoginCallbackFormPost = "/login/externalidp/callback/form"
+	EndpointSAMLACS                       = "/login/externalidp/saml/acs"
```

Comment:
the extra endpoint is required because of the saml middleware, right?

Reply:
exactly, it's because the saml middleware uses the provided rootURL"/ui/login/login/externalidp" as base and adds "/saml/acs"